netbox/core/models/jobs.py

```diff
@@ -216,23 +216,23 @@ def enqueue(cls, func, instance, name='', user=None, schedule_at=None, interval=
 
         return job
 
-    def trigger_webhooks(self, event):
-        from extras.models import Webhook
+    def trigger_events(self, event):
```

Comment:
The logic herein is quite similar to `extras.events.process_event_rules()`. We should see if it's possible to condense them somehow. (I'm also on favor of renaming this to explicitly reference the processing of event _rules_ and not just events.)